Torch.API/Managers/IPluginManager.cs

```diff
@@ -14,12 +14,12 @@ public interface IPluginManager : IManager, IEnumerable<ITorchPlugin>
         /// <summary>
         /// Fired when plugins are loaded.
         /// </summary>
-        event Action<IList<ITorchPlugin>> PluginsLoaded;
+        event Action<ICollection<ITorchPlugin>> PluginsLoaded;
 
         /// <summary>
         /// Collection of loaded plugins.
         /// </summary>
-        IList<ITorchPlugin> Plugins { get; }
+        IDictionary<Guid, ITorchPlugin> Plugins { get; }
```

Comment:
Again, `IReadOnlyDictionary`.  We don't want people using this to "unload" plugins.